content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/130-logging.mdx

```diff
@@ -112,4 +116,59 @@ Duration: 2ms
 </cmdResult>
 </CodeWithResult>
 
+</tab>
+
+<tab>
+
+<CodeWithResult expanded="{true}">
+<cmd>
+
+```ts highlight=4,5,22-25;normal
+const prisma = new PrismaClient({
+  log: [
+    {
+      emit: 'event',
+      level: 'query',
+    },
+    {
+      emit: 'stdout',
+      level: 'error',
+    },
+    {
+      emit: 'stdout',
+      level: 'info',
+    },
+    {
+      emit: 'stdout',
+      level: 'warn',
+    },
+  ],
+})
+
+prisma.$on('query', (e) => {
+  console.log('Query: ' + e.query)
+  console.log('Params: ' + e.params)
+  console.log('Duration: ' + e.duration + 'ms')
+})
+```
+
+</cmd>
+<cmdResult>
+
+```terminal
+Query: db.User.deleteMany({ _id: ( $in: [ “62261e0b18139c6099ba7097”, ], }, })
+Params: [0]
+Duration: 3ms
+Query: db.User.deleteMany({ _id: ( $in: [ “6226277a96069500743edcf9”, ], }, })
+Params: [2, 7, 18, 29]
+Duration: 3ms
```

Comment:
The two queries currently seem to include the params, that is probably not correct (or MongoDB logging works different than I would expect)

Reply:
Oops, thanks, that was a copy paste error (I haven't been able to test this yet). Removed the params for now

Reply:
both properties (duration and params) are passed but they will be `undefined`, see 
![image](https://user-images.githubusercontent.com/94194/157862198-252bd059-5f7a-4502-865b-839d62f34e1a.png)

Reply:
Ah, ok. I've removed both parameters from the example and added a note to the reference page that these are undefined in MongoDB.